MediaBrowser.Controller/Entities/Movies/Movie.cs

```diff
@@ -51,7 +51,7 @@ public override double GetDefaultPrimaryImageAspectRatio()
                 return 0;
             }
 
-            return 2 / 3;
+            return 2.0 / 3;
```

Comment:
It needs to be done in multiple classes.

Reply:
This fixes that specific bug. @Bond-009 will be submitting a PR to fix the remaining ones.